concepts/logic.coral

```diff
@@ -5,6 +5,10 @@ Logic components are defined by using the 'ssr' or 'logic' keyword. A logic
 component ALWAYS expects a return value of the Component type.
 **/
 
+<!-- CoralForm is a form component that prevents default form behavior -->
```

Comment:
```suggestion
// CoralForm is a form component that prevents default form behavior
```
**HTML comments should not be mixed with Coral comments.**

Reply:
Oops, this was because of some quick copy-pasting. 😅